libvcx/src/messages/agent_utils.rs

```diff
@@ -204,8 +206,10 @@ fn _create_issuer_keys(my_did: &str, my_vk: &str, my_config: &Config) -> VcxResu
 pub fn configure_wallet(my_config: &Config) -> VcxResult<(String, String, String)> {
     let wallet_name = get_or_default(&my_config.wallet_name, settings::DEFAULT_WALLET_NAME);
 
-    wallet::init_wallet(
+    wallet::create_and_open_as_main_wallet(
         &wallet_name,
+        &my_config.wallet_key,
+        &my_config.wallet_key_derivation.as_deref().unwrap_or("ARGON2I_INT".into()),
```

Comment:
The default KDF was extracted into a static `WALLET_KDF_DEFAULT`

Reply:
good catch